lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +27,40 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
```

Comment:
I just realized why this is done this way: because we set these attributes when we retrieve a course grade from the database. 

https://github.com/edx/edx-platform/pull/13804/files/50ae6c7648cd14a8719466eebb7b530c3f7bbdfd#diff-32687833c38e231b51a8f27a76c72a56R58